src/index.js

```diff
@@ -35,6 +35,9 @@ module.exports = function(config) {
  * @returns {{get: get, save: save, all: all, allById: allById}}
  */
 function getStorageObj(client, namespace) {
+    var delete = function(id, cb) {
```

Comment:
would you mind changing this to a function declaration (`function delete(id, cb) {...}`)? That avoids any possible problems introduced by hoisting.